### PR TITLE
Fix #1702: activate cancer case on viewing variants.

### LINKED ISSUE #1702: Scout cancer variant view does not activate case
As a scout cancer user, I would like my cases to activate when I peruse variants much as for RD cases.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**How to test**:
1. as a non-admin user, open a cancer case and visit the variants page
1. note that it does not activate
Screenshot 2020-02-12 at 10 40 02
1. apply this branch, and note that the same procedure now activates the case.
Screenshot 2020-02-12 at 10 40 38

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
PIng @christacosta: note that this will not work right away, but only after next release!

**Comment by northwestwitch** [comment]
Do you need a review?

**Comment by northwestwitch** [comment]
Works: 
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Cancer case HPO panel variants link
 - Fix so that some drop downs have correct size
 - First IGV button in str variants page
+- Cancer case activates on SNV variants
```
Comment (northwestwitch) [inline CHANGELOG.md]: What about SV variants? Because we have those as well now!

**Comment by dnil** [comment]
We should unify the views as well - there should be a common base variant view class, and some children for different kinds of variants. But, mañana!

File: scout/server/blueprints/variants/views.py [hunk 2]
```diff
@@ -277,6 +277,16 @@ def cancer_variants(institute_id, case_name):
         page = int(request.args.get("page", 1))
         form = CancerFiltersForm(request.args)
 
+    # update status of case if visited for the first time
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/views.py]: You could move this function to the controllers and re-use it from several other functions: 
- snv cancer variants
- sv cancer variants
- snv variants
- sv variants
- str variants? (current situation is that you don't activate the case when you look at these)

File: CHANGELOG.md (hunk 1 above)
Comment (dnil) [reply CHANGELOG.md]: Already working.

Comment (northwestwitch) [reply CHANGELOG.md]: Ah ok!

File: scout/server/blueprints/variants/views.py (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/variants/views.py]: Good catch on the STR variants - I'll add that to this PR!

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
...
+- Cases activate when STR variants are viewed
```
Comment (northwestwitch) [inline CHANGELOG.md]: 👍

File: scout/server/blueprints/variants/controllers.py [hunk 4]
```diff
@@ -787,3 +787,25 @@ def verified_excel_file(store, institute_list, temp_excel_dir):
             written_files += 1
 
     return written_files
+
+def activate_case(store, institute_obj, case_obj, current_user):
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/controllers.py]: 👍

**Pull request merged**